saltedge-compliance-connector-java-v2/compliance-connector-sdk/src/test/java/com/saltedge/connector/sdk/api/controllers/ais/TokensV2ControllerIntegrationTests.java

```diff
@@ -50,11 +48,10 @@
 /**
  * Tests Interceptors + TokensV2Controller
  */
-@RunWith(SpringRunner.class)
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 public class TokensV2ControllerIntegrationTests extends ControllerIntegrationTests {
 
-    @Before
+    @BeforeEach
```

Comment:
You don't use JUnit 5, so why replace @Before with @BeforeEach?

Reply:
i'm using JUnit 5. (testImplementation 'org.junit.jupiter:junit-jupiter-api:5.10.2')